frontend/src/__tests__/cypress/cypress/pages/pipelines/topology.ts

```diff
@@ -30,12 +30,36 @@ class PipelineRunBottomDrawer extends Contextual<HTMLDivElement> {
   }
 }
 
+class PipelineRunRightDrawer extends Contextual<HTMLDivElement> {
+  findRightDrawerInputOutputTab() {
+    return this.find().findByTestId('right-drawer-tab-Input / Output');
```

Comment:
I'm not sure if we have a guideline for this anywhere, but at least for consistency, ideally test IDs are kebab-case to be consistent with formatting we have elsewhere right now.

So this looks to be a matter of applying `toLowerCase`, along with probably a `replace` with regex to remove these empty spaces inside of `PipelineRunDrawerRightTabs.tsx`

Reply:
We had a mapping with another similar type so used that instead, and added the same for the run bottom drawer too.